### PR TITLE
Animated

### PR DESCRIPTION
**Pull request by alaa140101**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -208,8 +208,12 @@ Check out all of the submissions below! Well done everyone!
 - Ozziel Ruiiz - [Github](https://ozzielruiz.github.io/animated-template/)
 - Roy Wang - [Github](https://github.com/falinwang/portfolio), [View live](https://falinwang.github.io/portfolio/)
 - Hamza Khan [GitHub](https://github.io/salderahamza/page-animation) [Live](https://salderahamza.github.io/page-animation)
+<<<<<<< HEAD
```
Comment (MattCSmith) [inline README.md:211]: This line

File: README.md [hunk 2]
```diff
@@ -208,8 +208,12 @@ Check out all of the submissions below! Well done everyone!
...
+- Alaa Balbaid [GitHub](https://github.com/alaa140101/animated) [Live](https://alaa140101.github.io/animated/)
+=======
```
Comment (MattCSmith) [inline README.md:213]: This line

File: README.md [hunk 3]
```diff
@@ -208,8 +208,12 @@ Check out all of the submissions below! Well done everyone!
...
 - WayneMacMavis - [Github](https://github.com/waynemacmavis/animated-template), [Viewlive](https://waynemacmavis.github.io/animated-template/)
 - Carls13 - [Github](https://github.com/Carls13/20-javascript-projects/animated-template), [Viewlive](https://Carls13.github.io/20-javascript-projects/animated-template)
+>>>>>>> cad64e0f792086c8c32d44b98ae83784c5507cbc
```
Comment (MattCSmith) [inline README.md:216]: This line

**Review by MattCSmith — CHANGES_REQUESTED** [review]
Hey @alaa140101 
Any chance you can remove the few lines mentioned below, they do not need to be present
thanks

**Review by alaa140101 — COMMENTED** [review]
resolved

**Pull request merged**